README.md

```diff
@@ -290,37 +267,27 @@ If you use another Linux distribution please install the corresponding packages.
         make
         sudo make install
 
-#### `configure` options
-
-You can use these optional arguments to `./configure` for creating a Makefile:
+#### Options for `configure`
 
-    --no-fuse-group | --fuse-group (only COMMON)
-        Some distributions require user to be in group 'fuse' to use
-        sshfs and encfs. This toggles the check on or off.
-
-    --python3 | --python (all)
-        Use either 'python3' or 'python' to start Python Version 3.x
-
-Note: The first value is default.
-
-See also `common/configure --help` and `qt/configure --help`
-
-
-## NewsFeed
-
-Back In Time has an RSS feed
-https://feeds.launchpad.net/backintime/announcements.atom
+You can use these optional arguments to `./configure` for creating a Makefile.
```

Comment:
delete "these" since the list of options is missing now and must be fetched via cmd line